pkg/login/ldap.go

```diff
@@ -54,6 +54,17 @@ func (a *ldapAuther) login(query *LoginUserQuery) error {
 	if ldapUser, err := a.searchForUser(query.Username); err != nil {
 		return err
 	} else {
+
+		for _, ldapGroup := range a.server.LdapGroups {
```

Comment:
should there not a a condition for this group lookup? For all ldap installs that have memberOf